database/auth.go

```diff
@@ -98,8 +98,14 @@ func (db *Db) storeVestingAccount(account exported.VestingAccount) (int, error)
 			start_time = excluded.start_time
 			RETURNING id `
 
+	// Store the vesting account
+	err := db.SaveAccounts([]types.Account{types.NewAccount(account.GetAddress().String())})
```

Comment:
I see that `SaveVestingAccounts()` is called in `modules/auth/handle_genesis.go` as well where `SaveAccounts()` will also be called in the same block. I wonder if this line would better be moved to `cmd/parse/auth/vesting.go` if your intention is to also insert into `account` table for that parse command?

Reply:
Those two functions `SaveAccounts()` and `SaveVestingAccounts()` are only called when processing the genesis file and we want to first store all base account addresses inside `account` table because the `address` column inside the `vesting_address` table has a reference to `account` table.  I added this line `err := db.SaveAccounts([]types.Account{types.NewAccount(account.GetAddress().String())})` to ensure the vesting address is stored inside `account` table and avoid errors